mslib/msui/wms_control.py

```diff
@@ -1478,24 +1478,21 @@ def append_multiple_images(self, imgs):
         """
         images = [x for x in imgs if x]
         if images:
-            if hasattr(self.view, 'fig') is False:
```

Comment:
this was needed before not to crash on a different attribute than 'fig'

https://geoportal2.duisburg.de/arcgisserver/services/Luftbilder/Historische_Luftbilder/MapServer/WMSServer

now it crashes on 

```
Fatal error in MSS 8.3.0 on Linux-6.2.0-10018-tuxedo-x86_64-with-glibc2.35
Python 3.10.8 | packaged by conda-forge | (main, Nov 22 2022, 08:23:14) [GCC 10.4.0]

Please report bugs in MSS to https://github.com/Open-MSS/MSS

Information about the fatal error:

Traceback (most recent call last):
  File "/home/reimar/MAIN/MSS/mslib/msui/wms_control.py", line 1383, in continue_retrieve_image
    self.display_retrieved_image(img, legend_img, layer, style, init_time, valid_time, complete_level)
  File "/home/reimar/MAIN/MSS/mslib/msui/wms_control.py", line 1629, in display_retrieved_image
    self.view.draw_legend(self.append_multiple_images(legend_imgs))
  File "/home/reimar/MAIN/MSS/mslib/msui/wms_control.py", line 1484, in append_multiple_images
    max_height = int((self.view.plotter.fig.get_size_inches() * self.view.plotter.get_dpi())[1] * 0.99)
AttributeError: 'TopViewPlotter' object has no attribute 'get_dpi'
```


see also
https://github.com/Open-MSS/MSS/issues/1803